docs/metrics/metrics-dpm.md

```diff
@@ -0,0 +1,46 @@
+---
+id: metrics-dpm
+title: Metrics DPM
+sidebar_label: Metrics DPM
+description: Metrics DPM allows you to view metrics ingest volumes across your teams and services to find which sources are sending metrics data. 
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Data volume for metrics is measured in data points per minute (DPM). You can view your data volume for metrics by navigating to **Manage Data > Metrics > Metrics DPM**. The screen not only shows you which sources are sending metrics data, but how much, peaks, average, and credits used, as well as more granular information broken down by cluster, host, source, metrics cardinality, and reporting frequency. This data allows you to see which sources and metrics contribute most to your credit consumption.  
+
+It is important to understand the volume of metrics that you are ingesting into Sumo Logic, because when the DPM limit is exceeded, data is cached on the host and the source is throttled. For more information, see [Data Limits for Metrics](/docs/metrics/manage-metric-volume/data-limits-for-metrics/).
```

Comment:
```suggestion
Understanding the volume of metrics that you are ingesting is important because when you exceed the DPM limit, data is cached on the host and the source is throttled, reducing query performance and accuracy. For more information, see [Data Limits for Metrics](/docs/metrics/manage-metric-volume/data-limits-for-metrics/).
```